esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -102,25 +102,30 @@ Troubleshooting:
 //**********************//
 //*** WiFi settings ***//
 
+#define USE_BROADCAST_FOR_UDP // Comment out if you want to use only the first WiFi client to connect
```

Comment:
I understand the intention of the commenst etc. pp in this Wifi section, but I would like to keep the comments in this section more practical, i.e., more for illiterate folks like me. The following comments are along this line.

Pl move this to down below. Could e.g. go like
```
// for UDP (only UDP)
// Comment out if you want to use only the first WiFi client to connect
#define USE_BROADCAST_FOR_UDP
```

I'm not sure I understand the comment. Is it meaning Comment out if you want to connect to the first DHCP client? (here I pick up language used below)